File: 1-Beginner/1153/bee1153.java
// 1153 - Simple Factorial
import java.io.IOException;
import java.util.Scanner;

class bee1153 {

    public static void main(String[] args) throws IOException {
        Scanner scan = new Scanner(System.in);

        long n = scan.nextLong();

        for(long i = n-1; i > 1; i--) n *= i;

        System.out.println(n);

        scan.close();
    }
}

File: 1-Beginner/1153/bee1153.c
// 1153 - Simple Factorial
#include <stdio.h>
 
int main(void) {
 
    unsigned long long n; scanf("%llu", &n);
    
    for(int i = n-1; i > 1; i--) n *= i;
    
    printf("%llu\n", n);
 
    return 0;
}